rpc/internal/perf_service_test.go

```diff
@@ -284,6 +284,12 @@ func TestCreateMetricSeries(t *testing.T) {
 				assert.Equal(t, test.expectedResp.Id, resp.Id)
 				assert.Equal(t, test.expectedResp.Success, resp.Success)
 
+				conf, sess, err := cedar.GetSessionWithConfig(env)
```

Comment:
You should be able to just use the `GetDB` func here:
```
var result model.PerformanceResult{}
require.NoError(t, env.GetDB().Collection("perf_results").FindId(resp.Id).One(&result))
assert.False(t, result.Info.OverrideInfo.OverrideMainline)
```

I would also do this check after `CloseMetrics` is called and after we check the rollups (L299) just the field doesn't get set in any of those functions.